tests/testthat/test-HS_PGA_num.R

```diff
@@ -0,0 +1,83 @@
+test_that("check_clean_score", {
+  expect_equal(hs_pga_num(0, 0, 0), 1)
```

Comment:
For functions that take more than one input, can you please name all the inputs?  For example this would be:

```
hs_pga_num(abscess_fistula = 0, inflammatory_nodule = 0, non_inflammatory_nodule = 0)
```

That makes it easy to see what is supposed to be going in, and it makes it easier to debug issues in case of accidentally swapped argument order.

Please apply this throughout the code.

Reply:
Replaced with #12